pkg/auth/routes.go

```diff
@@ -236,6 +236,16 @@ func NewRouter(p *deps.RootProvider, configSource *configsource.ConfigSource) *h
 		// anonymous user
 		p.Middleware(newSettingsSubRoutesMiddleware),
 	)
+	webappPagePreviewChain := httproute.Chain(
+		rootChain,
+		p.Middleware(newPublicOriginMiddleware),
+		p.Middleware(newPanicWebAppMiddleware),
+		httproute.MiddlewareFunc(httputil.NoStore),
+		httproute.MiddlewareFunc(webapp.IntlMiddleware),
+		p.Middleware(newWebAppUIParamMiddleware),
```

Comment:
No need to use newWebAppUIParamMiddleware here.

Reply:
I think we need it since we need to support changing locale

Reply:
webapp.IntlMiddleware does that job.

Reply:
I removed for you.